README.md

```diff
@@ -622,13 +622,41 @@ Note: untrack doesn't get you out of the `notifying` state, which is maintained
 
 **Q**: What are Signals offering that `Proxy` doesn't currently handle?
 
-**A**: Proxies must wrap some object. They cannot be used to intercept property access/assignment to primitive values such as numbers, strings, or symbols.
-See Prior Implementations [tracked-built-ins](https://github.com/tracked-tools/tracked-built-ins/tree/master/addon/src/-private), [discussion](https://github.com/proposal-signals/proposal-signals/issues/101#issuecomment-2029802574). The following is valid for a Signal, but not for a Proxy:
+**A**: Proxies must wrap some object and do not inherently participate in a reactive graph. 
+Signals are things that can be part of this graph, and are well-behaved as citizens of it.
+The reactive graph is built over time by the relationships between reactive data, resulting in what is called 'autotracking'.
+
+Proxies also have a usage limitation: they cannot be used to intercept property access/assignment to primitive values such as numbers, strings, or symbols.
+ The following is valid for a Signal, but not for a Proxy:
 ```js
 new Proxy(0, { ... }) // TypeError: Cannot create proxy with a non-object as target or handler
 new Signal.State(0); // Perfectly valid
 ```
 
+Reactive objects are completely inert on their own and become special when used in a reactive _context_ that lets them autotrack dependencies.
+
+In this example, a reactive ui use case, the signal change will show up to the user, but the proxy change will not:
+```js
+const a = { count: 0 };
+const b = new Proxy(a, {});
+const c = new Signal.State(0);
+
+<template>
+  {b.count}
+  {c.get()}
+
+  <button onclick={() => {
+    b.count++;
+    c.set(c.get() + 1);
+  }}>change</button>
+</template>
+```
+when using a renderer that is optimized for fine-grained reactivity, clicking the button will cause the `c.get` cell to be updated, but `b.count` will forever remain 0, because `b` (and `b.count`) is not reactive.
```

Comment:
For the example above, yes, it is not reactive, but many frameworks implement their own abstraction around proxies to make them reactive. I think one of the value propositions from this proposal is the ability to define reactivity without having to create your own proprietary (framework specific) abstraction on top of proxies, plus the complexity of dealing with proxies, which is significant, when all you want is to know if the value is changing.

Reply:
> but many frameworks implement their own abstraction around proxies to make them reactive

ye, but then you wouldn't be doing `new Proxy(primitive)` -- this can never be reactive -- and maybe this is what I'm struggling to communicate -- Proxies have nothing to do with reactivity, ever, in any framework (even Vue!) they are a tool for abstracting away a less ergonomic underlying reactive primitive -- but not the reactive primitive themselves.

Any suggestions to help improve this would be most welcome <3

> on top of proxies,

under proxies? 

> the complexity of dealing with proxies, which is significant, when all you want is to know if the value is changing.

100% yes <3